BenchmarkVCFs/SimpleBenchmark.wdl

```diff
@@ -264,52 +267,126 @@ task VCFEval {
     command <<<
         set -xeuo pipefail
 
-        rtg format -o rtg_ref ~{reference.fasta}
-        rtg vcfeval \
-            ~{false="--all-records" true="" passing_only} \
-            ~{false="--squash-ploidy" true="" require_matching_genotypes} \
-            ~{true="--ref-overlap" false="" enable_ref_overlap} \
-            -b ~{base_vcf} \
-            -c ~{call_vcf} \
-            ~{"-e " + evaluation_bed} \
-            --vcf-score-field="~{score_field}" \
-            --output-mode split \
-            --decompose \
-            --roc-subset snp,mnp,indel \
-            -t rtg_ref \
-            -o output_dir \
+        # Normal situation without any PAR bed file
+        if [ -z ~{par_bed} ];
+        then
+            rtg format -o rtg_ref ~{reference.fasta}
+            rtg vcfeval \
+                ~{false="--all-records" true="" passing_only} \
+                ~{false="--squash-ploidy" true="" require_matching_genotypes} \
+                ~{true="--ref-overlap" false="" enable_ref_overlap} \
+                -b ~{base_vcf} \
+                -c ~{call_vcf} \
+                ~{"-e " + evaluation_bed} \
+                --vcf-score-field="~{score_field}" \
+                --output-mode split \
+                --decompose \
+                --roc-subset snp,mnp,indel \
+                -t rtg_ref \
+                -o reg \
+
+            mkdir output_dir
+            cp reg/*.vcf.gz* output_dir/
+
+        else
+            # Handle case where user provides PAR bed by running with --squash-ploidy over haploid region
+            grep -E '^@SQ' ~{reference.dict} | awk 'BEGIN{OFS="\t"} {gsub(/SN:/,"",$2); gsub(/LN:/,"",$3); print $2, $3}' > 'genome_file.txt'
+            bedtools complement -i ~{par_bed} -g genome_file.txt -L > 'par.bed'
+            bedtools complement -i par.bed -g genome_file.txt > 'reg.bed'
+
+            rtg format -o rtg_ref ~{reference.fasta}
+
+            rtg vcfeval \
+                ~{false="--all-records" true="" passing_only} \
+                ~{true="--ref-overlap" false="" enable_ref_overlap} \
+                --squash-ploidy \
+                -b ~{base_vcf} \
+                -c ~{call_vcf} \
+                --bed-regions par.bed \
+                ~{"-e " + evaluation_bed} \
+                --vcf-score-field="~{score_field}" \
+                --output-mode split \
+                --decompose \
+                --roc-subset snp,mnp,indel \
+                -t rtg_ref \
+                -o par
```

Comment:
similarly, would name this `non-par` instead (unless I'm just completely turned around right now, which is possible!)

Reply:
Yes!